src/components/Error/Error.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './Error.sass'
+
+const Error = () => {
+    return (
+        <section className='errorSection'>
```

Comment:
It'll be good to have this to use! I can see a few places where we might need to apply it.